### PR TITLE
Add ruleset: Bashy.io

### PR DESCRIPTION
**Pull request by dnoetzel**

### CODE REVIEW

File: src/chrome/content/rules/Bashy.io.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+ <!--
+	Nonfunctional hosts in *.bashy.io:
+		- bashy.io
```
Comment (J0WI) [inline src/chrome/content/rules/Bashy.io.xml]: Could be redirected to `www`.

**Comment by dnoetzel** [comment]
`https://launchpage.bashy.io/` gives a mixed content warning, because it tries to load resources from `http://www.opensourcebash.org/`, but since that domain is down, no content is actually loaded from an insecure resource. With that in mind, do you have any recommendations on how the `launchpage` host should be handled?

**Comment by J0WI** [comment]
Even if the resources would exists, it seems to be passive content only. You can ignore it.

File: src/chrome/content/rules/Bashy.io.xml [hunk 2]
```diff
@@ -1,13 +1,14 @@
- <!--
+<!--
 	Nonfunctional hosts in *.bashy.io:
-		- bashy.io
```
Comment (J0WI) [inline src/chrome/content/rules/Bashy.io.xml]: Please keep this comment to document the special rule below.

**Comment by J0WI** [comment]
Thanks!

**Comment by dnoetzel** [comment]
Thank you for all of the help. Sorry for all of the back and forth this PR has required.

**Pull request merged**